pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.flake8]
+# TODO add python directory here in later PR
+filename = ["./elements/**/*.py", "./question-servers/**/*.py"]
+ignore = ["E501"]
+# Black compatibility
+max-line-length = 88
```

Comment:
`.prettierrc.json` uses 100. Should we use 100 for consistency with that? I don't feel strongly about it, though.

Reply:
Black does have the ability to change this (one of the few configurable things), so I think changing to 100 lines isn't unreasonable.

Reply:
Let's keep this at 88 for now.